addons/MagicaVoxelImporter/VoxelMesh.gd

```diff
@@ -25,8 +28,15 @@ func _set_neck_height(value):
 		for mat in _get_mats():
 			mat.set_shader_param('neck_height', value)
 
-func _set_render_head(value):
-	render_head = value
-	if mesh:
-		for mat in _get_mats():
-			mat.set_shader_param("render_head",value)
+func set_render_head(value):
+	if is_inside_tree():
+		render_head = value
```

Comment:
always want to set the backing variable even if we aren't in the tree.. this should go before the if

Reply:
Done